CHANGES.rst

```diff
@@ -13063,4 +13063,4 @@ astropy.wcs
 0.1 (2012-06-19)
 ================
 
-- Initial release.
+- Initial release.
```

Comment:
Not sure why it is showing the difference while there's none.
Probably because I ran `towncrier --draft --version 4.3` for testing purposes, but I reverted the changes as soon as the files were changed.

Reply:
I suspect that it's something very silly, for example, there are several unicode characters that look like "-" in the font the github uses to display the code on this website; in fact they look the same in almost all fonts I've ever looked at. The easiest way to get rid of this is to manually edit the CHANGES.rst file in your tree. Remove this line and copy and paste the old line (from this website from locally when you open the CHANGES.rst in the main branch in a text editor) in. Git does not actually track when a file was edited, it only compares the file content character by character. If a file has exactly the same characters, git will assume it never changed.

Then, you squash the commits. That way, any knowledge of what happened to the file in between in erased and all is well again.

Reply:
Done. thanks!